package.json

```diff
@@ -75,7 +75,8 @@
   },
   "dependencies": {
     "@babel/runtime": "7.1.2",
-    "cornerstone-math": "^0.1.7"
+    "cornerstone-math": "^0.1.7",
+    "debug": "^4.1.1"
```

Comment:
Can you lock both of these to a specific version? (kill the `^`)

Reply:
Done